exercises/concept/chess-game/.docs/introduction.md

```diff
@@ -62,7 +62,7 @@ These methods when for example want to get the sum of all the values in the rang
 ## Endless & Beginless ranges
 
 A range can be endless and beginless.
-The endless or beginless range has there start or end value being `nil`, but when defining the range so can nil be omitted.
+The endless or beginless range has their start or end value being `nil`, but when defining the range so can nil be omitted.
```

Comment:
This file looks very much out of place here.

Reply:
what do you mean?

Reply:
This is coming from the `chess-game directory.  Looks out of place for a "New Concept: Range" patch.

Reply:
it is the range exercise

Reply:
Right, I see.  So the range concept (as per the title) and using the Chess game to show it off.  OK.